CHANGES.md

```diff
@@ -1,5 +1,9 @@
 ## Changelog
 
+### 2.4.2
```

Comment:
2.4.1 is the current in development release. Can you move these changes down to 2.4.1?